Makefile

```diff
@@ -97,7 +97,42 @@ run-app-bg:
 # fails then emit container logs before fast-failing the makefile target.
 .PHONY: test-run-app-dev
 test-run-app-dev:
-	docker compose down
+	mkdir -p /tmp/_conbench-promcl-coord-dir
+	docker compose down --remove-orphans
 	docker compose -f docker-compose.yml -f docker-compose.dev.yml \
 		up --build --wait --detach || (docker compose logs --since 30m; exit 1)
 	docker compose down
+
+
+# The version string representing the current checkout / working directory. The
+# default `dev` suffix represents a local dev environment. Override
+# CHECKOUT_VERSION_STRING with a different suffix (e.g. `ci`) in the CI
+# environment so that the version string attached to build artifacts reveals
+# the environment that the build artifact was created in.
+export CHECKOUT_VERSION_STRING ?= $(shell git rev-parse --short=9 HEAD)-dev
+DOCKER_REPO_ORG ?= conbench
+CONTAINER_IMAGE_SPEC=$(DOCKER_REPO_ORG)/conbench:$(CHECKOUT_VERSION_STRING)
```

Comment:
This is an attempt towards standardizing on globally applicable conbench web application container image naming. I don't think we upload to the Docker container hub (or something comparable) yet, but maybe we will. As long as we do not upload container images for the public to consume we can of course iterate on this naming convention.

The two make targets below use `CONTAINER_IMAGE_SPEC`.